containers/kubernetes/how-to/manage-allowed-ips.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta:
+  title: Managing allowed IP addresses for Kubernetes products
+  description: Learn how to manage allowed IP addresses for your Kubernetes Kapsule and Kosmos clusters. Configure access restrictions with our step-by-step guide.
+content:
+  h1: Managing allowed IP addresses for Kubernetes products
+  paragraph: Learn how to manage allowed IP addresses for your Kubernetes Kapsule and Kosmos clusters. Configure access restrictions with our step-by-step guide.
+tags: kubernetes kapsule kosmos
+dates:
+  validation: 2024-11-05
+  posted: 2024-11-05
+categories:
+  - containers
+---
+
+Restricting IPs on Kubernetes Kapsule or Kosmos clusters enhances security by limiting access to only trusted sources, thereby reducing the risk of unauthorized access and potential attacks.
+This control ensures that only specific IP addresses or networks can interact with your clusters, providing an additional layer of protection.
+The default entry `0.0.0.0/0` enables any host to establish a connection.
```

Comment:
```suggestion
Restricting IPs on Kubernetes Kapsule or Kosmos clusters enhances security by limiting access to only trusted sources. Since only IP addresses and networks specified by you can interact with your clusters, you reduce the risk of unauthorized access and potential attacks.

The default entry `0.0.0.0/0` enables any host to establish a connection.
```